client/src/components/OnboardingOne/OnboardingOne.css

```diff
@@ -0,0 +1,58 @@
+.OnboardingOne {
+  font-family: var(--default-font);
+}
+.OnboardingOne__form {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+.OnboardingOne__header {
+  margin: 0;
+}
+.OnboardingOne__container {
+  margin: 1rem 0;
+}
+.OnboardingOne__button-container {
+  display: flex;
+  justify-content: center;
+}
+.OnboardingOne .InputTextField__container--default {
+  height: 1rem;
+  width: 22.5rem;
+}
+
+.OnboardingOne .Button {
+  width: 18.875rem;
+  height: 4.07rem;
+  font-size: 2.25rem;
+  padding: 0;
+  margin: 2rem 0;
+}
+
+@media all and (max-width: 320px) {
+  .OnboardingOne__container {
+    flex-direction: column;
+    display: flex;
+    margin: 0.5rem 0;
+  }
+  .OnboardingOne__header {
+    font-size: 1.5rem;
+  }
+  .OnboardingOne .InputTextField__container--default {
+    min-width: 11.5rem;
+    max-width: 11.5rem;
+    font-size: 1rem;
+    height: 0.5rem;
+    padding: 0.7rem 1.4rem;
+  }
```

Comment:
You shouldn't modify the styling of a separate component like this. This will make debugging really difficult later. Instead, if that component isn't doing something you need, then you need to update that component